common/persistence/sql/sqlplugin/postgres/postgres_server_test.go

```diff
@@ -30,12 +32,17 @@ import (
 )
 
 const (
-	testUser      = "postgres"
-	testPassword  = "cadence"
 	testSchemaDir = "schema/postgres"
 )
 
 func getTestClusterOption() *pt.TestBaseOptions {
+	testUser := "postgres"
+	testPassword := "cadence"
+
+	if runtime.GOOS == "darwin" {
```

Comment:
Does it have to specify the os?

Reply:
Yes, because on OSX the default for postgresql user is USER which is not the case on other OSes, eg. on Ubuntu the default is "postgres" regardless of user.